Sidney-/slugtrade
# Fixed startup smoke test

This test is intended to verify that the environment is set up correctly--if this test fails, it must mean there's something wrong with the environment (rather than the app).

New functionality:

- Changed name of test_title to test_logo_loads.
- Changed test to check for the logo image in the navigation bar instead of checking the title of the page.

How to test:

- Run the script run_tests.sh and verify that the output contains the line "test_logo_loads: PASS".

Note:

- This test will fail if the server is not running.
- This test will fail if the virtual environment has not been installed correctly.
